### DISCUSSION TITLE
Luna vs npm create tamagui?

### DISCUSSION THREAD

**Discussion by junaid33** [criszz77/luna#85] [Q&A]
Hey, great work on the Luna template. I'm looking to use tamagui and was wondering what was the difference between your template and the official tamagui one?

**Comment by LunatiqueCoder** [comment]
Hello @junaid33 

Thank you for trying it out! While there are subtle differences, the most important ones are the following:

- There is no monorepo setup. Everything is one app, just like you can use Expo for native and web apps concurrently. In my opinion, you will develop faster and with less errors.
- Expo-router is not implemented yet. But that doesn't mean it can't be implemented. I'm currently working on that, but anyone should be able to add it by theirselves. 

You can use `app.native` folder for expo-router, and `app` folder for Next.js with the following babel config:


```javascript
process.env.EXPO_ROUTER_APP_ROOT = '../../app.native';
process.env.TAMAGUI_TARGET = 'native';

module.exports = function (api) {
  api.cache(true)
  return {
    presets: [['babel-preset-expo', { jsxRuntime: 'automatic' }]],
    plugins: [
      require.resolve('expo-router/babel'),
      // if you want reanimated support
      'react-native-reanimated/plugin',
      [
        '@tamagui/babel-plugin',
        {
          config: './tamagui',
          components: ['tamagui'],
        },
      ],
      [
        'transform-inline-environment-variables',
        {
          include: ['TAMAGUI_TARGET','EXPO_ROUTER_APP_ROOT'],
        },
      ],
    ],
  }
}
```
